### ISSUE TITLE
Gradle Error with Hello World project

### ISSUE THREAD

**Issue by bpeterman** [NativeScript/nativescript-dev-webpack#408]
I'm getting the following error when trying to use webpack for the Hello World app:
```
Gradle build...
         + applying user-defined configuration from /Users/bpeterman/workspace/HelloWorld/app/App_Resources/Android/app.gradle
         + adding nativescript runtime package dependency: nativescript-optimized
         + adding aar plugin dependency: /Users/bpeterman/workspace/HelloWorld/node_modules/tns-core-modules-widgets/platforms/android/widgets-release.aar
false
Running full build

FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':asbg:generateBindings'.
> No /Users/bpeterman/workspace/HelloWorld/platforms/android/build-tools/android-static-binding-generator/bindings.txt was found after runAstParser task was ran! Check to see if there are any .js files inside /Users/bpeterman/workspace/HelloWorld/platforms/android/app/src/main/assets/app
```
Here are my steps to reproduce (I'm targeting angular v4.4):

1. tns create HelloWorld --template nativescript-template-ng-tutorial
2. cd HelloWorld/
3. npm install --save-dev nativescript-dev-webpack@0.8.0
4. npm install
5. Downgrade to Angular versions ~4.4.0 and rxjs ~5.3.0
6. tns run android --bundle

Git repo of project: https://github.com/bpeterman/NSHelloWorld

Using these docs:
- https://docs.nativescript.org/best-practices/bundling-with-webpack#installation-and-configuration
- https://docs.nativescript.org/angular/tutorial/ng-chapter-1

package.json: 
```
{
  "description": "NativeScript Application",
  "license": "SEE LICENSE IN <your-license-filename>",
  "readme": "NativeScript Application",
  "repository": "<fill-your-repository-here>",
  "nativescript": {
    "id": "org.nativescript.HelloWorld",
    "tns-android": {
      "version": "3.4.1"
    }
  },
  "dependencies": {
    "@angular/animations": "~4.4.0",
    "@angular/common": "~4.4.0",
    "@angular/compiler": "~4.4.0",
    "@angular/core": "~4.4.0",
    "@angular/forms": "~4.4.0",
    "@angular/http": "~4.4.0",
    "@angular/platform-browser": "~4.4.0",
    "@angular/platform-browser-dynamic": "~4.4.0",
    "@angular/router": "~4.4.0",
    "nativescript-angular": "~4.4.0",
    "nativescript-theme-core": "~1.0.4",
    "reflect-metadata": "~0.1.8",
    "rxjs": "~5.3.0",
    "tns-core-modules": "~3.4.0",
    "zone.js": "~0.8.18"
  },
  "devDependencies": {
    "@angular/compiler-cli": "~4.4.0",
    "@ngtools/webpack": "~1.6.0",
    "babel-traverse": "6.26.0",
    "babel-types": "6.26.0",
    "babylon": "6.18.0",
    "copy-webpack-plugin": "~4.0.1",
    "extract-text-webpack-plugin": "~3.0.0",
    "lazy": "1.0.11",
    "nativescript-css-loader": "~0.26.0",
    "nativescript-dev-typescript": "~0.6.0",
    "nativescript-dev-webpack": "^0.8.0",
    "nativescript-worker-loader": "~0.8.1",
    "raw-loader": "~0.5.1",
    "resolve-url-loader": "~2.1.0",
    "typescript": "~2.4.2",
    "webpack": "~3.2.0",
    "webpack-bundle-analyzer": "^2.8.2",
    "webpack-sources": "~1.0.1"
  },
  "scripts": {
    "ns-bundle": "ns-bundle",
    "publish-ios-bundle": "npm run ns-bundle --ios --publish-app",
    "generate-android-snapshot": "generate-android-snapshot --targetArchs arm,arm64,ia32 --install",
    "start-android-bundle": "npm run ns-bundle --android --run-app",
    "start-ios-bundle": "npm run ns-bundle --ios --run-app",
    "build-android-bundle": "npm run ns-bundle --android --build-app",
    "build-ios-bundle": "npm run ns-bundle --ios --build-app"
  }
}

```

**Comment by bpeterman** [comment]
Is anyone else getting this issue?

**Comment by bpeterman** [comment]
@sis0k0 do you have any idea why this might be happening?  I've gotten this in multiple projects.

**Comment by gizmo385** [comment]
@sis0k0 Any updates on this ticket? This is completely preventing our team from using Webpack.

**Comment by sis0k0** [comment]
Heya! The `tns run android --bundle` command is available from v0.9 of the nativescript-dev-webpack plugin. If you want to use older version of the plugin, instead of the above command you will have to run the npm scripts that were added to your package.json after installing the plugin. For example - `npm run start-android-bundle`.

Here's how your dependencies should look like when using different versions of the plugin:
```
Command for bundling - `npm run start-android-bundle`
nativescript-dev-webpack - v0.8.0
NativeScript CLI - v3.3.0
Android runtime version (tns-android) - v3.3.0
tns-core-modules - v3.3.0
nativescript-angular- v4.4.0
@angular/core (and other angular packages) - v4.4.0

Command for bundling - `tns run android --bundle`
nativescript-dev-webpack - v0.9.0+
NativeScript CLI - v3.4.0+
Android runtime version (tns-android) - v3.4.0+
tns-core-modules v3.4.0+
nativescript-angular - v5.0.0+
@angular/core (and other angular packages) - v5.0.0+
```
